src/classes/CON_DoNotContact_TEST.cls

```diff
@@ -29,255 +29,1346 @@
 */
 /**
 * @author Salesforce.org
-* @date 2015
+* @date 2020
 * @group Contacts
 * @group-content ../../ApexDocContent/Contacts.htm
-* @description Handles changes to the deceased and do not contact fields on Contact.
+* @description Tests to validate the functionality of the CON_DoNotContact_TDTM class to
+* synchronize Household naming exclusions and communication preferences with changes to the Deceased
+* and Do Not Contact fields for the Contact object.
 */
 @isTest
 private class CON_DoNotContact_TEST {
+    /**************************************************************************************************************************
+    ****************************************************** FUNCTIONAL TESTS ****************************************************
+    **************************************************************************************************************************/
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that contacts inserted with "Deceased" = true are excluded from 
+    * from Household naming functionality and have the following fields automatically marked as true: 
+    * Do Not Contact, Email Opt Out, Do Not Call, Do Not Fax
+    */
+    @isTest 
+    private static void insertContactDeceasedIsTrue(){
+        List<Contact> testContactsList = UTIL_UnitTestData_TEST.getMultipleTestContacts(5);
+        
+        for (Contact con : testContactsList){
+            con.Deceased__c = true;
+        }
+
+        Test.startTest();
+        insert testContactsList;
+        Test.stopTest();
+
+        List<Contact> contactsAfterInsertList = [SELECT Do_Not_Contact__c, 
```

Comment:
In the future we may want to consider using a format that allows us to see the query in context with a single indentation similar to `[
    SELECT Do_Not_Contact__c
    ....
    ....
    FROM CONTACT
    WHERE Id IN :testContactsList];
]`

We're using longer variable names now, which means our SOQL query indentation is pretty deep in the format displayed in this PR. Just a thought for future discussion.